### ISSUE TITLE
React Native version mismatch

### ISSUE THREAD

**Issue by jpadula** [viromedia/figment-ar#6]
Hello there,

I'm trying to run the project using Viromedia app to test. 

I get "React Native version mismatch" ->

Javascript version: 0.55.1
Native version: 0.57.1

I tried to upgrade react-native to 0.57.1 and I get errors just with VRX files (I checked the rn-cli.config.js and its included). -> Unable to resolve module 'XXXXXXXXXX.vrx'

any thoughts?

**Comment by mattrencher** [comment]
@jpadula having React Native Version mismatch issue as well. Before that Figment was running very well from Xcode, I was able to drop my own models in very easily. Not sure what changed last week, but I can't even get Figment to run now.

**Comment by jpadula** [comment]
Yes, I was able to run before, but now I cant even if I change the RN Version
